.circleci/config.yml

```diff
@@ -34,6 +34,8 @@ workflows:
           filters:
             branches:
               only: /(^renovate-.*|^nori/.*)/
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
```

Comment:
**issue**: would you be able to revert this particular change? we have an [open bug](https://financialtimes.atlassian.net/browse/CPP-1428) for this but Tool Kit shouldn't overwrite it if you delete it now. not exactly sure what the cause is or whether it's still present in the latest Tool Kit version but we can work that out when someone picks up the ticket.